page.php

```diff
@@ -16,14 +16,14 @@
 get_header();
 ?>
 
-  <div id="content" class="site-content <?= bootscore_container_class(); ?> py-5 mt-5">
+  <div id="content" class="site-content <?= apply_filters('bootscore_container_class', 'container', 'page'); ?> py-5 mt-5">
     <div id="primary" class="content-area">
 
       <!-- Hook to add something nice -->
       <?php bs_after_primary(); ?>
 
       <div class="row">
-        <div class="<?= bootscore_main_col_class(); ?>">
+        <div class="<?= apply_filters('bootscore_main_col_class', 'col'); ?>">

```

Comment:
This may create issues, because `col` isn't enough. If main content has for example a large image, `col` takes the entire width of the container and pushes sidebar down. It has to checked if a sidebar is active or not. See this very first PR https://github.com/bootscore/bootscore/pull/7.

Reply:
Great 👍